test/parallel/test-module-create-require-from-directory.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const path = require('path');
+
+const { createRequireFromPath } = require('module');
+
+const p = path.join(path.resolve(__dirname, '..', 'fixtures'), '/');
```

Comment:
I'll update this test to use `path.sep` so the build will succeed on a windows platform.

Reply:
Does this still need to be added?